apps/wallet-connect/public/manifest.json

```diff
@@ -1,8 +1,11 @@
 {
-  "short_name": "WalletConnect",
   "name": "WalletConnect",
   "description": "Allows your Gnosis Safe Multisig to connect to dapps via WalletConnect.",
-  "iconPath": "wallet-connect.svg",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "icons": [
+    {
+      "src": "wallet-connect.svg",
+      "sizes": "any",
```

Comment:
is `any` a part of standard?

Reply:
I extracted it from here:

- https://developer.mozilla.org/en-US/docs/Web/Manifest/icons (Link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-sizes)

So I think so, but taking advantage of your comment I'm going to look up as well for `any` in the `purpose` prop

Reply:
Awesome. Also wanted to share that the spec from w3c sometimes contradicts MDN: https://www.w3.org/TR/appmanifest/

I remember this from my adventures with manifests in the past 😄

Reply:
Yeah I looked as well to the spec and cannot see anything related to the any value. But still I think is really used